app/Http/Controllers/HomeController.php

```diff
@@ -15,6 +18,52 @@ public function __construct()
     {
         $this->middleware('auth');
     }
+    public function recentTransactions()
+    {
+        $data = array();
+        //2 months ago
+        $data[0] = Transaction::where('date', '>=',  new Carbon('first day of last month'))
+            ->where('date', '<=', new Carbon('last day of last month'))
+            ->sum('total_amount');
+
+        //most recent month
+        $data[1] = Transaction::where('date', '>=', new Carbon('first day of this month'))
+            ->where('date', '<=', Carbon::now())
+            ->sum('total_amount');
+
+        return $data;
+    }
+
+    public function recentCustomerVolume()
+    {
+        return Customer::all()->count();
+    }
+
+    public function totalSalesValue()
+    {
+        $storeSalesPrevious = [];
+        $storeSalesCurrent = [];
+        $multiStoresSalesDifference = [];
+        $rights = Auth::User()->GetRights();
```

Comment:
I think this should be Auth::user()->GetRights();

With a lowercase u

Reply:
sure, see update